src/pages/Stats/StatsView.jsx

```
@@ -0,0 +1,316 @@
+                    fontSize: '14px',
+                    maxWidth: '320px',
+                    lineHeight: '1.5',
+                }}>
+                    Use the search above to find players or teams and view their detailed statistics
+                </div>
+            </div>
+        );
+    }
+
+    // Memoize win percentage calculation
+    const winPercentage = useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)